tests/resources/test_resources.py

```diff
@@ -64,6 +64,16 @@
         "active": True,
         "start_datetime": datetime(2022, 7, 20, 20, 0, 0).strftime("%Y-%m-%d %H:%M:%S"),
     },
+    "everywhere_banner": {
+        "message": "Everywhere banner",
+        "url_path": "",
+        "category": "info",
+        "active": True,
+        "start_datetime": datetime(2022, 7, 20, 20, 0, 0).strftime("%Y-%m-%d %H:%M:%S"),
+        "end_datetime": (datetime.utcnow() + timedelta(days=20)).strftime(
```

Comment:
Since `utcnow` is [deprecated already](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow), should we avoid using it? Maybe use `datetime.now(tz=timezone.utc).replace(tzinfo=None)` instead?